src/main/java/io/kestra/plugin/microsoft365/sharepoint/Create.java

```diff
@@ -0,0 +1,180 @@
+package io.kestra.plugin.microsoft365.sharepoint;
```

Comment:
We should add a sub package `drive` here since all your tasks handle files or folders on a Drive inside a given Site.

Then I'll create another PR to handle also SharePoint's sites and lists in dedicated packages.

Reply:
All tasks are concerned to be moved inside a new `.drive` package